src/maestro/agents/beeai_agent.py

```diff
@@ -288,6 +290,23 @@ async def _create_agent(self):
             tools.append(sandbox_tool)
             self.print(sandbox_tool.name)
 
+        for tool in self.agent_tools:
+            if tool.lower() not in [
```

Comment:
rather than creating a giant list here we should save the above lists as vars and use a concat of the lists here instead, that would only define each tool name once.

Also it may be worth revisiting those hard-coded tool names. I originally added those as hard-coded to match the various names I found in both our test/demos/doc and the ones in beeai, but it may be better to have one documented name per internal tool to minimize potential overlap with map tool names